code/modules/clothing/masks/gasmask.dm

```diff
@@ -72,7 +72,18 @@
 
 /obj/item/clothing/mask/gas/explorer/folded/Initialize()
 	. = ..()
-	adjustmask()
+	forceadjustmask()
+
+/obj/item/clothing/mask/gas/explorer/folded/proc/forceadjustmask()
```

Comment:
```suggestion
/obj/item/clothing/mask/gas/explorer/folded/proc/forceadjustmask()
```
could you make this use snake case? `force_adjust_mask()`

Reply:
done